### ISSUE TITLE
Cross-Plattform Issues when comparing text

### ISSUE THREAD

**Issue by alexanderfink** [oblador/loki#324]
Hello, 

I have the issue, that when comparing reference images from Windows on macOS, the text renderings have slight differences like thicker lines (see images). 

![button_current](https://user-images.githubusercontent.com/30551988/125576791-8b9f1c59-869e-460a-a517-a5a0f2d09ea6.png)
![button_ref](https://user-images.githubusercontent.com/30551988/125576821-7bb8ed8d-2fae-4e86-857a-2573a4eafb0a.png)
![button_diff](https://user-images.githubusercontent.com/30551988/125576897-5c1fd1c4-2b64-4e67-8e92-1371a152deae.png)

We use the following configuration: 
```
configurations: {
    'chrome.laptop': {
      target: 'chrome.app',
      width: 1366,
      height: 768,
    },
  },
```

I used to have a similar problem when comparing colors on macOS because Chrome used another Color-System. Forcing it to use sRGB via `chrome://flags/` fixed this for me. Maybe there is something compared to this for text-rendering as well? 

Thanks in advance!

**Comment by techeverri** [comment]
> I have the issue, that when comparing reference images from Windows on macOS

Using the `chrome.app` target only works consistently if you generate the baselines in the same OS/laptop

### How to fix it

One of the key features of Loki is to have "Reproducible tests independent of OS" to achieve this you have to use the "Chrome in docker" target.

🔗 https://loki.js.org/configuration.html#configurations

Example 👇 

https://github.com/oblador/loki/blob/8beda16fcda4841c4e23c7353880e94498cbadcc/examples/react/loki.config.js#L4-L9

_**TL;DR:** Loki will run against a fixed version of Chromium within Docker_
Don't forget to check the [optional dependencies](https://loki.js.org/getting-started.html#optional-dependencies)
